concept-wip/numbers/about.md

```diff
@@ -0,0 +1,249 @@
+# Numbers
+
+Crystal has a type `Number` which is the base type for all numeric types.
+Inside of the Number type there are two main types: `Int` and `Float`.
+Int is short for integer and represents whole numbers.
+Int can be both negative and positive, and doesn't have a decimal point.
+
+```crystal
+1.class
+# => Int32
+
+-1.class
+# => Int32
+```
+
+Float represents numbers with a decimal point, floats can be both negative and positive.
+
+```crystal
+10.5.class
+# => Float64
+
+1.0.class
+# => Float64
+```
```

Comment:
Below in `hints.md` you mention to not worry about specific types and to just use `Int` and `Float`, but is it worth mentioning that the `32` and `64` shown here indicate the size? It might be confusing to have the `class` of these literals not match `Int` and `Float` without any mention of why this is.